### DISCUSSION TITLE
Could an API be added to read the "mouse"

### DISCUSSION THREAD

**Discussion by erlendnagel** [paulscottrobson/neo6502-firmware#284] [Q&A]
The firmware appears to support a mouse device, but there is no API function to access the status of the mouse from the 6502. This would be quite convenient to have for the simple drawing program i am thinking about creating. (the nunchuck will be an acceptable substitute until then)

**Comment by paulscottrobson** [comment]
Sascha Schneider is doing a revamp of the USB which is primarily about the gamepad, but will include mouse, once that is done, then yes.

**Comment by astralaster** [comment]
#409 added mouse support to the API.
